### ISSUE TITLE
Rename SignUp to Register or CreateAccount

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/lucky_cli#456]
SignUp and SignIn are very similar and it takes a moment to figure out what they are doing, especially for non-native English speakers. We should use a different term for "creating an account". I think a common one is "Register" or just "Create Account"

* `SignUps::New/Create` -> `Registrations::New/Create`
* `SignUpUser` -> `RegisterUser`

**Comment by jwoertink** [comment]
Sort of related https://github.com/luckyframework/lucky/issues/1003. If we get rid of the auto generated routes, then it won't be an issue, but with them we run in to issues with `/signins` -> `/logins`, etc..

**Comment by paulcsmith** [comment]
Yeah good point, I think we should start moving the generators to manually defined routes to start out with! That should be an "easy" change

**Comment by stephendolan** [comment]
@jwoertink I was having a bit of trouble following from the previous title through all the comments, but it sounds like we landed on using the terms `LogIn` and `SignUp`, right?

**Comment by stephendolan** [comment]
Yeah, I haven't really heard a peep in Discord about this lately, but I guess that our English-speaking chat kind of naturally selects against the non-English-speaking confusion Paul cited in the original description.

**Comment by jwoertink** [comment]
Along with this, I'd also like to change the name of the default Layouts. Using `AuthLayout` to say you're not logged in, as where `MainLayout` is where you are logged in really only seems to work if your site has no public pages aside from the sign-in/up pages... (oddly enough, one of my Lucky apps fits that exact description lol). For me, I think better names might be `MainLayout`, and `AppLayout`. OR... we could just swap them.. `AuthLayout` meaning you're Authenticated / Authorized... and MainLayout just being your main one... 

What do you think @stephendolan ?

**Comment by stephendolan** [comment]
@jwoertink I actually think the fact that we could use it for both of these purposes is enough of a reason to nix AuthLayout altogether.

I remember when I used a web framework (Rails) for the very first time, AppLayout and GuestLayout just made sense to me. There's probably a great reason Paul didn't go with GuestLayout, but I think it's the most discoverable and newbie-friendly term to use.

**Comment by jwoertink** [comment]
It came up for me because I was writing the tutorial, and I wrote this bit:

> The `MainLayout` is currently setup to be our site layout for users that are logged in to the app.
>  The `AuthLayout` is used for pages like the login/signup pages, and now our Home Page.

And I had to instruct the user to update the home page to inherit from`AuthLayout` instead of `MainLayout`. I felt it was confusing to explain. 

I'm not that great with naming things, but I do feel like this one in particular could be a weird confusion for new people.

**Comment by grepsedawk** [comment]
I agree, AuthLayout is confusing
To me, it sounds like "The layout that is only displayed when user is auth'd"

**Comment by splashsky** [comment]
+1 for `AppLayout` and `GuestLayout`, as previously suggested by Jeremy and Stephen. This is coincidentally the same naming scheme [used by the Laravel Breeze package](https://github.com/laravel/breeze/tree/1.x/stubs/default/resources/views/layouts) for it's auth bootstrapping.

When not bootstrapping auth, `MainLayout` is a perfectly fine name.
